ads/adcrops.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, checkData, validateDataExists} from '../3p/3p';
+
+const adcropsFields = ['adcropsjs_params'];
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function adcrops(global, data) {
+  checkData(data, adcropsFields);
+  validateDataExists(data, adcropsFields);
+
+  global.adcropsjs_params = JSON.parse(data.adcropsjs_params);
```

Comment:
You will need to change this to
`global['adcropsjs_params']` to make our linter happy.